src/Resources/config/services.yml

```diff
@@ -101,3 +101,29 @@ services:
             - [setAnnotationReader, ['@annotations.cached_reader']]
             - [setContainer, ['@service_container']]
     Umanit\SeoBundle\SchemaOrg\SchemaOrgResolver: '@umanit_seo.schema_org.schema_org_resolver'
+
+    # Block Resolver
```

Comment:
This has no place here, please revert the commit `a1926226fbba53af462310bcec63a4b10bf628ad`